admin-dev/themes/new-theme/public/theme.rtlfix

```diff
@@ -1,54 +1,57 @@
 .sidebar.sidebar-right {
-  -webkit-transform: translate(-100%, 0);
-  -moz-transform: translate(-100%, 0);
-  -ms-transform: translate(-100%, 0);
-  -o-transform: translate(-100%, 0);
-  transform: translate(-100%, 0);
+    -webkit-transform: translate(-100%, 0);
```

Comment:
Please, do not change the whole indentation :/

Reply:
The reason is that I copied all the codes into my PHPStorm and it changed the whole indentations. Is it necessary to undo them?

Reply:
Yes, we use 2 spaces for indentation in css for this project.
It also helps to see what is the exact change.